src/apps/relay/netengine.c

```diff
@@ -350,6 +350,10 @@ void set_ssl_ctx(ioa_engine_handle e, turn_params_t *params) {
     params->tls_ctx_update_ev = ev;
     TURN_MUTEX_UNLOCK(&turn_params.tls_mutex);
   }
+  else
+  {
```

Comment:
args is a parameter to the function,

are you certain that all users of the function are expecting the parameter to be freed by this function?

Reply:
I believe that the new event added has callback update_ssl_ctx with EV_PERSIST. Therefore,  it doesn't stop being pending when its callback runs. The event can be triggered more than once because it is on persist mode. We can only free this args if the event is not created.

Reply:
@jonesmz Thank you for reviewing my PRs! I was wondering if this one still looks good. Also I am unaware of the process to merge the PR into the repository after it has been accepted. Thanks for all the help!

Reply:
Hi, you're welcome.

I am not able to merge PRs.

Basically you just need to wait for one of the persons with the ability to merge PRs to decide to merge it.

They might ask for additional modifications, or they might merge it as it is.